app/controllers/submissions_controller.rb

```diff
@@ -205,13 +205,13 @@ def collect_and_begin_grading
     grouping = Grouping.find(params[:id])
 
     if assignment.submission_rule.can_collect_grouping_now?(grouping)
-      # Push grouping to the priority queue
-      SubmissionCollector.instance.push_grouping_to_priority_queue(grouping)
+      grouping.update(is_collected: true)
+      grouping.save
```

Comment:
I'm don't think we want to set this state here.  I think it would be better to set it in the function that actually does the collecting.